index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>ToDo List</title>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <p id="demo">Input task and click on the button to populate the list</p>
+    <div class="searchLayout">
+      <div class="taskFieldWrapper">
+        <div class="taskField">
+          <label for="text">Enter your task</label>
+          <input type="text" name="text" id="text" placeholder="Enter Task" />
+        </div>
+        <button id="btn" class="searchButton" onclick="btnSubmit()">
+          Click Me
+        </button>
+      </div>
+      <div class="searchField">
+        <label for="search">Search:</label>
+        <input
+          type="text"
+          name="search"
+          id="search"
+          placeholder="Search here..."
+        />
+      </div>
+    </div>
+    <ul id="list"></ul>
+    <div class="buttons">
+      <button id="save" class="FooterButton" onclick="saveDeal()">Save</button>
+      <button id="revive" class="FooterButton" onclick="retrieveDeal()">
```

Comment:
remove this, item should get saved when added and should get loaded when page refreshed.